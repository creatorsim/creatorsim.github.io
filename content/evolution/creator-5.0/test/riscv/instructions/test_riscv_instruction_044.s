
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
    main:
      li  x5, 882
      li  x6, 881
      li  x7, 5
      li  x8, 543
      ble x5, x6, jump1
      ble x7, x8, jump1

    jump2:
      li t3, 34
      li a7, 10
      ecall

    jump1:
      li x5, 11
      li x6, 555
      ble x5, x6, jump2
