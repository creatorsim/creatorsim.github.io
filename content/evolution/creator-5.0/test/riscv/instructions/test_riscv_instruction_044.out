
cr[PC]:0x24; ir[x5,t0]:0xb; ir[x6,t1]:0x22b; ir[x7,t2]:0x5; ir[x8,fp,s0]:0x21f; ir[x17,a7]:0xa; ir[x28,t3]:0x22; keyboard[0x0]:''; display[0x0]:''; 

